src/zino/state.py

```diff
@@ -40,7 +40,7 @@ def dump_state_to_file(self, filename: str):
         """Dumps the full state to a file in JSON format"""
         _log.debug("dumping state to %s", filename)
         with open(filename, "w") as statefile:
-            statefile.write(self.model_dump_json(exclude_none=True, indent=2))
+            statefile.write(self.model_dump_json(exclude_none=True, indent=2, by_alias=True))
```

Comment:
I have to admit, I don't understand the point of this at all.  The state file needs no backwards compatibility with Zino 1 at all, so any field name aliases are completely irrelevant here (@stveit and @johannaengland )

Reply:
When looking at this today I realized the same...

Reply:
So, resolved by #353 then, I guess :)